_projects/nested-languages.md

```diff
@@ -24,16 +24,106 @@ tags:
   - Core
   - Bears
 ---
-Multiple programming languages can coexist in a single source file. coala would
+Generally, a single source file contains code for exactly one programming language.
+But multiple programming languages can also coexist in a single source file. coala would
 benefit from the ability to determine the programming language of each part of
-a file. If we have e.g. a PHP file, it is likely that we'd have chunks of html
+a file. If we have e.g. a PHP file, it is likely that we'd have chunks of HTML
 inside a few of them. We would like coala to be able to still run meaningful
 analysis on those files.
 Other examples include:
 
-* awk inside bash
-* JSON inside Javascript
-* HTML, LaTeX or others inside Jinja2
+* PHP inside Javascript
+
+PHP is a server side script (Runs at server side) while
+JavaScript is a client side scripting i.e when a request is made your browser
+(client) runs the Java script, so it is impossible to run PHP inside
+Javascript, but we can run Javascript inside PHP.
+
+For example
+
+Suppose if you want to write something in Javascript. Then it is necessary
+that the Javascript code should be in PHP file i.e. if file name is `temp.PHP`
+then Javascript code should be in script tag.
+
+eg. `temp.php` (your PHP filename)
+
+```
+  <html>
+  <head>
+  <script>function myFunc(){
+  alert('<?php echo date("Y/m/d"); ?>');
```

Comment:
why on earth would anyone do this?

what is the purpose of putting php in javascript inside html?

do people really do this?

nested languages are read problems, and very hard problems - no need to create your own ficticious ones.

go back and read the issue again.  you were given good examples.

these examples are worse than the ones in the published version.

any more nonsense and i'll close the PR, because this needs to be written up properly as it is a very important project for next year.

Reply:
okay sorry @jayvdb  I think I should do more research about this concept ,but are all examples which I mentioned in the proposal are unsuitable??